README.md

```diff
@@ -1,3 +1,98 @@
-# excubiae
+# Excubiae
 
-an on-chain flexible &amp; modular framework for implementing custom gatekeepers.
+Excubiae is a composable framework for building custom attribute-based access control policies on Ethereum.
+
+The framework provides a set of common, abstract, and flexible smart contracts to simplify the creation of reusable, attribute-based criteria. These contracts, called "gatekeepers," are solution-agnostic, enforcing checks against user-provided evidence and maintaining records of those who meet the criteria.
+
+This approach enables seamless interoperability across different protocols. For instance, a single check could combine verifiable attributes from Semaphore and MACI, ensuring flexible and composable access control based on two different protocols. Indeed, for example, you can define criteria to verify token ownership or/and validate a zero-knowledge proof (ZKP). Using these criteria, you can create a policy to enforce the checks and integrate it seamlessly into your smart contract logic. A practical use case might involve requiring verification before registering a new voter for a poll (e.g., in a MACI-based voting system).
+
+You can learn more in this [design document](https://hackmd.io/@0xjei/B1RXoTh71e).
```

Comment:
suggestion: If you are curious to dig deeper into Excubiae, you can learn more[..]